docs/running-local.md

```diff
@@ -0,0 +1,236 @@
+Running On a Local (Big) Machine
+================================
+
+Prerequisites
+-------------
+
+You need a beefy Linux `x86_64` machine, with `docker` and `docker-compose`
+available (and ready to use by the current UNIX user).
+You'll also need to access this host on port 443 (HTTPS).
+
+For example on Ubuntu 16.04 (LTS):
+
+    sudo apt-get install -y docker docker-compose
+    sudo adduser $USER docker
+    sudo su $USER # so that the group is taken into account
+    docker ps # should not complain :)
+
+Get Secotrec-local
+------------------
+
+Then we get the `secotrec-local` application:
```

Comment:
@iskandr: this was what I was mentioning during our afternoon meeting: this particular document covers lots of bases and therefore hard to understand for epidisco's target readers; so I would consider this as a complete detailed document and something we will encourage developers/more-tech-oriented people to take a look at this.

The rest should go somewhere else to start, a simple page that explains the contexts and introduces them: [Kubernetes](https://kubernetes.io/) does this reasonably well, also [TensorFlow](https://www.tensorflow.org/), even [Flask](http://flask.pocoo.org/), or [GCloud](https://cloud.google.com/sdk/): let users land on a page with lots of overview and then let hem explore or dive deep. 

Here the "quick-start" part is a bit lost due to all the introduction and alternative scenerios, but it really takes 10-15 minutes to get all up and running and now only with a few commands. I would love to help get such a page up and running and then we will make sure it is more appealing.